pkg/command/context.go

```diff
@@ -1000,16 +1000,18 @@ func displayContextListOutputListView(cfg *configtypes.ClientConfig, writer io.W
 				context = ctx.ClusterOpts.Context
 			}
 		}
-		op.AddRow(ctx.Name, ctx.Target, isMgmtCluster, isCurrent, ep, path, context)
+
+		op.AddRow(ctx.Name, ctx.Target, strconv.FormatBool(isMgmtCluster), strconv.FormatBool(isCurrent), ep, path, context, ctx.AdditionalMetadata)
```

Comment:
Necessary to maintain same output behavior as prior to the tanzu-plugin-runtime update.